### PR TITLE
WeEth base onboarding - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/WeETHAaveV3BaseOnboarding.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+[Ether.fi](http://ether.fi/) has also launched eETH on Arbitrum, allowing users on the L2 to get exposure to the LRT yield and points. As weETH has already been approved for onboarding to Aave v3 Ethereum and Arbitrum by the DAO, this proposal aims to extend the onboarding of weETH to Aave v3 on Base.
+
+## Specification
+
+The table below illustrates the configured risk parameters for **weETH**
+
+| Parameter                          |                                      Value |
+| ---------------------------------- | -----------------------------------------: |
+| Isolation Mode                     |                                       true |
```
Comment (rustboyar) [inline src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/WeETHAaveV3BaseOnboarding.md:25]: should be false here

File: src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/WeETHAaveV3BaseOnboarding.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+| Flashloanable                      |                                    ENABLED |
+| Siloed Borrowing                   |                                   DISABLED |
+| Borrowable in Isolation            |                                   DISABLED |
+| Oracle                             | 0xFc4d1d7a8FD1E6719e361e16044b460737F12C44 |
+
+## References
+
+- Implementation: [AaveV3Base](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol)
+- Tests: [AaveV3Base](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.t.sol)
+- [Snapshot](TODO)
```
Comment (rustboyar) [inline src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/WeETHAaveV3BaseOnboarding.md:55]: you can remove this line

File: src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/WeETHAaveV3BaseOnboarding.md (hunk 1 above)
Comment (marczeller) [reply src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/WeETHAaveV3BaseOnboarding.md]: fixed.

Interesting same yarn generate issue happened but wasn't detected for https://github.com/bgd-labs/aave-proposals-v3/pull/266/files#diff-e764ceeec86af20be4f5b784f3c72458aa392de79ab962ed9958eff99849fa4fR42

@sakulstra

File: src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/WeETHAaveV3BaseOnboarding.md (hunk 2 above)
Comment (marczeller) [reply src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/WeETHAaveV3BaseOnboarding.md]: done.

File: src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      eModeCategory: AaveV3BaseEModes.ETH_CORRELATED,
+      enabledToBorrow: EngineFlags.ENABLED,
+      stableRateModeEnabled: EngineFlags.DISABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
+      withSiloedBorrowing: EngineFlags.DISABLED,
+      flashloanable: EngineFlags.ENABLED,
+      ltv: 72_50,
+      liqThreshold: 75_00,
+      liqBonus: 7_50,
+      reserveFactor: 45_00,
```
Comment (rustboyar) [inline src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol:43]: i see 15% in recommendations

File: src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+      supplyCap: 150,
+      borrowCap: 30,
+      debtCeiling: 0,
+      liqProtocolFee: 10_00,
+      rateStrategyParams: IV3RateStrategyFactory.RateStrategyParams({
+        optimalUsageRatio: _bpsToRay(35_00),
```
Comment (rustboyar) [inline src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol:49]: 45%

File: src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol (hunk 3 above)
Comment (marczeller) [reply src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol:43]: https://governance.aave.com/t/arfc-adjusting-interest-rate-curve-for-weeth-on-arbitrum-and-base/17804

File: src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol (hunk 4 above)
Comment (marczeller) [reply src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol:49]: https://governance.aave.com/t/arfc-adjusting-interest-rate-curve-for-weeth-on-arbitrum-and-base/17804

File: src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol (hunk 3 above)
Comment (rustboyar) [reply src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/AaveV3Base_WeETHAaveV3BaseOnboarding_20240527.sol:43]: make sense to add the link to this post to AIP text to specification section

**Review by rustboyar — APPROVED** [review]

**Review by kyzia551 — APPROVED** [review]

**Pull request merged**